src/main/java/uk/gov/hmcts/reform/cpo/service/impl/CasePaymentOrdersServiceImpl.java

```diff
@@ -1,9 +1,110 @@
 package uk.gov.hmcts.reform.cpo.service.impl;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
 import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.cpo.data.CasePaymentOrderEntity;
+import uk.gov.hmcts.reform.cpo.exception.CasePaymentOrdersQueryException;
+import uk.gov.hmcts.reform.cpo.repository.CasePaymentOrderQueryFilter;
+import uk.gov.hmcts.reform.cpo.repository.CasePaymentOrdersRepository;
 import uk.gov.hmcts.reform.cpo.service.CasePaymentOrdersService;
+import uk.gov.hmcts.reform.cpo.service.mapper.CasePaymentOrderMapper;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
 
 @Service
 public class CasePaymentOrdersServiceImpl implements CasePaymentOrdersService {
 
+    @Autowired
+    private CasePaymentOrderMapper casePaymentOrderMapper;
+    @Autowired
+    private CasePaymentOrdersRepository casePaymentOrdersRepository;
+
+    public CasePaymentOrdersServiceImpl(CasePaymentOrderMapper casePaymentOrderMapper, CasePaymentOrdersRepository casePaymentOrdersRepository) {
+        this.casePaymentOrderMapper = casePaymentOrderMapper;
+        this.casePaymentOrdersRepository = casePaymentOrdersRepository;
+    }
+
+    public Page<CasePaymentOrderEntity> getCasePaymentOrders(final CasePaymentOrderQueryFilter casePaymentOrderQueryFilter) {
+        if (casePaymentOrderQueryFilter.isItAnEmptyCriteria()) {
+            return Page.empty();
+        }
```

Comment:
maybe better to have this in the controller

Reply:
TODO